README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- Kindly update your ``README`` File following the requested information by the template.
<img width="1041" alt="Screen Shot 2021-11-02 at 18 33 52" src="https://user-images.githubusercontent.com/22258274/139969941-0b23af95-aff2-43c7-ae9d-b2f2a062d7d4.png">